src/status_im/contexts/settings/saved_addresses/view.cljs

```diff
@@ -0,0 +1,41 @@
+(ns status-im.contexts.settings.saved-addresses.view
+  (:require [quo.core :as quo]
+            [quo.theme :as quo.theme]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.resources :as resources]
+            [status-im.contexts.settings.saved-addresses.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn empty-state
+  []
+  (let [theme (quo.theme/use-theme)]
+    [quo/empty-state
+     {:title           (i18n/label :t/no-saved-addresses)
+      :description     (i18n/label :t/you-like-to-type-43-characters)
+      :image           (resources/get-themed-image :sweating-man theme)
+      :container-style style/empty-container-style}]))
+
+(defn view
+  []
+  (let [insets          (safe-area/get-insets)
+        saved-addresses []]
+    [quo/overlay
+     {:type            :shell
+      :container-style (style/page-wrapper (:top insets))}
+     [quo/page-nav
+      {:key        :header
+       :background :blur
+       :icon-name  :i/arrow-left
+       :on-press   #(rf/dispatch [:navigate-back])}]
```

Comment:
you can define this outside, there's similar fns in the file (doesn't matter now since saved-addresses is not a sub, but once you make it a sub `page-nav` will re-render if saved-addresses changes)

Reply:
probably we could move this to some namespace something like `:on-press nav/navigate-back` because it's used in many places

Reply:
Not just navigate back, also navigate to, open modal and other common handlers.

Reply:
I'll mark this resolved because this discussion is beyond the scope of this PR.